01_fib/fibonacci.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import "fmt"
+
+func fib(n int) {
+
+	var (
+		fn int
+		f0 int
+		f1 = 1
+	)
+
+	for i := 1; i <= n; i++ {
+		if i <= 1 {
+			fn = 1
+		} else {
+			fn = f0 + f1
+			f0 = f1
+			f1 = fn
+		}
+		fmt.Println(fn)
+	}
+}
+
+func main() {
+	fib(7)
```

Comment:
Not sure what the task is, but it would good to output the result somewhere.

Reply:
@anzdaddy , the task suggests to hard code.
Create an executable go program in directory 01_fib
Write a function that prints the first n Fibonacci numbers
func fib(n int)
**Call fib(7) in main to print**